lib/command/promote_app_from_upstream.rb

```diff
@@ -32,7 +32,7 @@ def copy_image_from_upstream
     def deploy_image
       args = []
       args.push("--run-release-phase") if config.current[:release_script]
-      run_cpflow_command("deploy-image", "-a", config.app, *args)
+      run_cpflow_command("deploy-image", "-a", config.app, "--use-digest-ref", *args)
```

Comment:
Maybe we should provide `use_digest_ref` option for `promote-app-from-upstream` command too - user may not want this behaviour enabled by default